fixing rel=0 in cms-element-youtube-video

lietzi opened the pull request:
The second videoUrl in the concatenated string disabled the rel=0 configuration.

<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/community/contribution-guideline?category=shopware-platform-dev-en/community).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?
The option "rel=0" configures the embeded youtube-player to show related videos only from the own channel. With the videoUrl in front this option is ignored and the list of related videos contains any kind of youtube videos.

### 2. What does this change do, exactly?
It activates the rel=0 option to only list videos from the own channel.

### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

janbuecker commented:
This change introduces some other problems. If the attribute is already created, things get saved correctly. If you don't have an attribute for the dispatch yet, it will result in endless new attribute entries. This is because the data sent from the server does not match with the mapping in ExtJS - in fact, the answer loses the attributeId and dispatchId.

Since we are implementing a whole new attribute management in Shopware 5.2, we won't fix this issue.

FiveDigital commented:
If the new attribute system enables saving of dispatch attributes it is okay for us :+1: Otherwise I could fix this PR. Since you are doing this in the PaymentController, too it might not be too expensive.

See: https://github.com/shopware/shopware/blob/5.1/engine/Shopware/Controllers/Backend/Payment.php#L183 and https://github.com/shopware/shopware/blob/5.1/engine/Shopware/Controllers/Backend/Payment.php#L241

janbuecker commented:
Yup, it will. :-)

Keeping this PR closed. Thank you

FiveDigital commented:
I'm really excited about the new attribute management and appreciate platform changes more than new marketing features :+1:

lietzi committed:
fixing rel=0

The second videoUrl in the concatenated string in line 11 disabled the rel=0 configuration.

mitelg commented:
thanks for you contribution @lietzi  :+1: :tada: :blue_heart:
